srcpkgs/pipewire/template

```diff
@@ -18,13 +18,13 @@ configure_args="
  -Drlimits-match=@_pipewire
  $(vopt_feature ffado libffado)
 "
-hostmakedepends="doxygen graphviz pkg-config python3-docutils gettext glib-devel"
-makedepends="$(vopt_if sdl2 SDL2-devel) gst-plugins-base1-devel jack-devel
- sbc-devel v4l-utils-devel libva-devel libbluetooth-devel ncurses-devel
- libfreeaptx-devel libusb-devel fdk-aac-devel libsndfile-devel Vulkan-Headers
- vulkan-loader pulseaudio-devel avahi-libs-devel webrtc-audio-processing-devel
- readline-devel openssl-devel lilv-devel libcanberra-devel dbus-devel
- libmysofa-devel opus-devel $(vopt_if ffado libffado-devel)"
+hostmakedepends="doxygen gettext glib-devel graphviz pkg-config python3-docutils"
+makedepends="avahi-libs-devel dbus-devel fdk-aac-devel gst-plugins-base1-devel
+ jack-devel libbluetooth-devel libcanberra-devel $(vopt_if ffado libffado-devel)
+ libfreeaptx-devel liblc3-devel libmysofa-devel libsndfile-devel libusb-devel
+ libva-devel lilv-devel ncurses-devel openssl-devel opus-devel pulseaudio-devel
+ readline-devel sbc-devel $(vopt_if sdl2 SDL2-devel) v4l-utils-devel
+ Vulkan-Headers vulkan-loader webrtc-audio-processing-devel"
```

Comment:
do we need to sort these? it's going to make blame a pain

Reply:
Fair enough, I'll just add it to the end.